core/src/main/java/feast/core/service/StatsService.java

```diff
@@ -129,6 +129,9 @@ public GetFeatureStatisticsResponse getFeatureStatistics(GetFeatureStatisticsReq
         List<FeatureNameStatistics> featureNameStatistics =
             getFeatureNameStatisticsByDate(
                 statisticsRetriever, featureSet, features, timestamp, request.getForceRefresh());
+        if (featureNameStatistics.size() != 0) {
+          featureNameStatisticsList.add(featureNameStatistics);
```

Comment:
Why isnt this in `getFeatureNameStatisticsByDate`?

Reply:
Would you mean returning null if the size is 0 in `getFeatureNameStatisticsByDate`?